### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [HalfSweet/Blog#20] [comment]
# about/

## 自我介绍
* 我是半糖，19岁，事学生
* 爱好电子，~~精通~~(看过)多种单片机使用方法以及编程语言
* 最喜欢的屏幕是墨水屏
* 计划学习更多有关ee和cs的内容，并希望今后从事类似的工作

https://blog.halfsweet.cn/about/

**Comment by ddzzkunye** [comment]
直接把个人信息挂在网上啊, 不怕被线下吗,  
毕竟RLE.wiki的数据是在大佬这里啊
